.github/workflows/build_and_test.yaml

```diff
@@ -53,29 +52,68 @@ jobs:
         name: codecov-envoy-gateway
         verbose: true
 
-    # build
-    - name: Build Multiarch EG Binaries
+  build:
+    runs-on: ubuntu-latest
+    needs: [lint, gen-check, license-check, coverage-test]
+    steps:
+    - uses: actions/checkout@v3
+    - uses: ./tools/github-actions/setup-deps
+
+    - name: Build EG Multiarch Binaries
```

Comment:
great stuff, thoughts on uploading the container image rather than binary itself, so we dont need to build the image 3 times before running each conformance test ?
this doesnt need to be tackled in this PR, but would be good to track it with a GH issue

Reply:
This could be tracked in a new issue, thanks for the tips.